libs/platform/src/lib/components/select/select.component.ts

```diff
@@ -77,13 +78,12 @@ export class SelectPlatformComponent extends fdSelect implements OnInit, AfterCo
      */
     private _value: any;
 
-    constructor(private cd: ChangeDetectorRef, private rtl: RtlService) {
-        super(cd, rtl);
-        this.fdDropdownClass = false;
+    constructor(private cd: ChangeDetectorRef, elementRef: ElementRef) {
+        super(elementRef, new SelectProxy(), cd);
```

Comment:
We need change this but this is just a comment. - we need to get rid of this inheritance.